Makefile

```diff
@@ -2,26 +2,29 @@
 .DEFAULT_GOAL := help
 
 test: # Run Unit Test
-	python -m unittest tests
+	@python3 -m unittest tests
 
 test_email: # Test Send Email
-	python -m unittest tests.TestSendEmail
+	@python3 -m unittest tests.TestSendEmail
 
 build: # Build distribution for SDK
-	python setup.py sdist bdist_wheel
+	@python3 setup.py sdist bdist_wheel
 
 publish: # Publish the package to PyPI
-	python -m twine upload dist/*
+	@python3 -m twine upload dist/*
```

Comment:
It's fine to use `python` since it will use the one inside the venv, which should be v3.